### ISSUE TITLE
[Q] how to get max value of chart

### ISSUE THREAD

**Issue by besbesmany** [wandb/client#2428]
![W B Chart 7_21_2021, 9_42_17 PM](https://user-images.githubusercontent.com/35992924/126550187-43372524-8219-4493-900b-0f9293ccad83.png)

**Comment by vanpelt** [comment]
In your script after `wandb.init` has been called, you can add the following:

```
wandb.run.define_metric("metrics/mAP_0.5", summary="max")
```

That will then add a column in your workspace named "metrics/mAP_0.5.max" which you can sort or filter by.

**Comment by besbesmany** [comment]
can I do it from https://wandb.ai/ directly without specific code in colab?

I didn't write any wandb code with yolov5 I just run this

`import wandb
`
`!python train.py --img 416 --batch 16 --epochs 100 --data data.yaml --weights yolov5s.yaml --name yolov5s_results  --cache --bbox_interval 30 --save_period 30 --device 0`

what I should use other than `wandb.init`?

is there detailed examples to have full control on chart in https://wandb.ai/ site?

**Comment by vanpelt** [comment]
You would need to modify `train.py` to add the line described after calling `wandb.init`.

**Comment by besbesmany** [comment]
I added `--upload_dataset` to `!python train.py`
I have this chart
can I  add `("metrics/mAP_0.5", summary="max") ` like in chart `runs.summary["validation dataset"]`

![5](https://user-images.githubusercontent.com/35992924/126681515-09af77d3-d4e1-403a-bf6a-ba0d4efae894.png)

**Comment by vanpelt** [comment]
@besbesmany the instructions I gave you are for modifying your local python files, not making changes in the wandb interface.  You would need to add `wandb.run.define_metric("metrics/mAP_0.5", summary="max")` to `train.py` after calling `wandb.init`.

For your "validation dataset" wandb.Table shown here, you can sort by any of the metrics logged in the table to find the individual examples that have the highest or lowers mAP.

**Comment by besbesmany** [comment]
how to show training dataset and its accuracy in wandb

**Comment by besbesmany** [comment]
@vanpelt if you please give me script example for yolov5
with 
`wandb.init`
`wandb.run.define_metric("metrics/mAP_0.5", summary="max"`

or where to add them in train.py

**Comment by vanpelt** [comment]
https://github.com/ultralytics/yolov5/blob/master/train.py#L106

Add the define_metric call after the line above.  No need to call wandb.init

**Comment by besbesmany** [comment]
than you so much :)
